package.json

```diff
@@ -20,7 +20,7 @@
     "test": "node scripts/run-tests.js",
     "lint": "pnpm run --recursive lint && pnpm prettier --check",
     "lint:fix": "pnpm run --recursive lint:fix && pnpm prettier --write",
-    "prettier": "prettier *.md \"{docs,.github}/**/*.{md,yml,ts,js}\" \"scripts/**/*.js\""
+    "prettier": "prettier *.md \"{book,docs,.github}/**/*.{md,yml,ts,js}\" \"scripts/**/*.js\""
```

Comment:
We specifically don't want to do this to improve readability and minimise diff sizes (we write one sentence per line)